install/Sidebar.shtml

```diff
@@ -1,5 +1,4 @@
-<!-- Sidebar.shtml -->
-<!-- This is the common sidebar definition for /install pages -->
+jmri.or<!-- This is the common sidebar definition for /install pages -->
```

Comment:
Is it intended to have `jmri.or` before the comment?

Reply:
No, that's a mistake.  Thanks for catching.